odoc.opam

```diff
@@ -25,7 +25,7 @@ delimited with `(** ... *)`, and outputs HTML.
 """
 
 depends: [
-  "odoc-parser" {>= "0.9.0"}
+  "odoc-parser" {>= "0.9.0" & < "1.0.0"}
```

Comment:
I don't think we should keep this. The CI will be fixed and this will break pinning Odoc.

Reply:
I'd gladly revert this if there's some other way we can make ocaml-ci pass - but it's way to easy for breakage to slip in if we can't. The master branch is currently broken because we ignored the failure from ocaml-ci thinking it was just the formatting failure, when in fact there were other tests failing.

Reply:
I'm not sure. It's easy to check for the CI and I don't expect it to stay broken for long. I don't think it's a big deal if the build is broken on master, because we'll notice quickly. On the other hand, it's easy to forget to revert this.